features/poller/monitors/core/src/test/java/org/opennms/netmgt/poller/monitors/ActiveMQMonitorTest.java

```diff
@@ -0,0 +1,102 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2017 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2017 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.netmgt.poller.monitors;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.activemq.broker.BrokerService;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.Rule;
+import org.junit.rules.TestName;
+import static org.junit.Assert.*;
+import org.junit.Before;
+import org.opennms.core.utils.InetAddressUtils;
+import org.opennms.netmgt.poller.MonitoredService;
+import org.opennms.netmgt.poller.PollStatus;
+import org.opennms.netmgt.poller.support.SimpleMonitoredService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ *
+ * @author roskens
+ */
+public class ActiveMQMonitorTest {
+    private static final Logger LOG = LoggerFactory.getLogger(ActiveMQMonitorTest.class);
+    private BrokerService broker;
```

Comment:
The ActiveMQBroker unit doesn't let you specify a connector for the broker. I've enhanced it to let you pass a broker URL and it will add that as a connector for the broker.

Fixed.